src/content/protocols/pendle.md

```diff
@@ -0,0 +1,433 @@
+
+The `PendleFeeDistributorV2Proxy` functions as the distribution contract for the collected fees that are rewarded to vePENDLE holders for their respective voting power. Under the `DevMultisig` (2/3) administration, this contract employs merkle proofs through `setMerkleRootAndFund` to set the claims while simultaneously depositing funds for claiming.
+
+The `hardwareDeployer` EOA maintains additional protocol functionalities with its `GUARDIAN` and `ALICE` roles in the `PendleGovernanceProxy`. These capacities include fee rate modifications through `setLnFeeRate` and operational controls.
+
+In parallel, the legacy `PendleFeeDistributor` remains active under the control of an unidentified address (0xD9c9935f4BFaC33F38fd3A35265a237836b30Bd1). This contract continues to distribute USDC to users through a separate reward system not accountable to the current Treasury structure.
+
+### PENDLE Emissions to LPs
+
+The PENDLE token emission system operates through an interconnected contract architecture with dynamic reward distribution.
+
+The emission cycle begins when the `PendleLiquidityIncentivesMultisig` claims newly minted PENDLE through `claimLiquidityEmissions` based on a predetermined schedule. The PENDLE token's parameters are protected by a 7-day timelock via `initiateConfigChanges`. These tokens then flow to `Governance`, which funds the `PendleGaugeControllerMainchainUpg` through the `fundPendle` function.
+
+vePENDLE holders lock their PENDLE in the `VotingEscrowPendleMainchain` contract and direct incentives by voting on pool allocations through `vote(pools, weights)` in the `PendleVotingControllerUpg`. This contract verifies vePENDLE balances and manages configuration parameters including emission rates (`setPendlePerSec`) and pool eligibility (`addPool`, `removePool`).
+
+The `PendleVotingControllerUpg` sends voting results to the `PendleGaugeControllerMainchainUpg` via `updateVotingResults`, which then allocates PENDLE rewards to `PendleMarketV3` contracts dynamically.
+
+Finally, LPs can claim their accumulated rewards directly from `PendleMarketV3` contracts via `redeemRewards`, completing a system where incentives are continuously allocated in response to liquidity activity and governance decisions.
+
+### Points Rewards
+
+The `PendleMultiTokenMerkleDistributor` manages token points system rewards for vePENDLE holders through a monthly cycle. The system takes a snapshot of vePENDLE balances on the 20th of each month, with token distributions occurring approximately one week later. The `DevMultisig` (2/3) controls this contract.
+
+The process uses the `setMerkleRoot` function to establish which addresses can claim which rewards, `upgradeToAndCall` to modify the implementation, and `transferOwnership` to transfer administrative control.
```

Comment:
this sentence (line 224) does not make much sense, first part is good but

add additional sentences for the following information

`upgradeToAndCall` to modify the implementation, and `transferOwnership` to transfer administrative control.